testomania/src/main/java/com/earth/testomania/apis/opentdb/domain/models/OpenTDBApiCategory.kt

```diff
@@ -0,0 +1,27 @@
+package com.earth.testomania.apis.opentdb.domain.models
+
+enum class OpenTDBApiCategory(val id: Int) {
+    GENERAL(9), //TODO Not sure that we need this
+    BOOKS(10),
+    FILM(11),
+    MUSIC(12),
+    MUSICALS_AND_THEATRES(13), //TODO Not sure that we need this
+    TELEVISION(14),
+    VIDEO_GAMES(15),
+    BOARD_GAMES(16),
+    NATURE(17),
+    COMPUTERS(18), //TODO Not sure that we need this
+    MATHEMATICS(19),
+    MYTHOLOGY(20),
+    SPORTS(21),
+    GEOGRAPHY(22),
+    HISTORY(23),
+    POLITICS(24),
+    ART(25),
+    CELEBRITIES(26),
+    ANIMALS(27),
+    VEHICLES(28),
+    COMICS(29),
+    GADGETS(30),
+    ANIME_MANGA(31), //TODO Not sure that we need this
```

Comment:
How can we not need Anime? :o

Reply:
Ok lets include it

Reply:
Yeah I mean, thats what a lot of kids like so...